common/djangoapps/student/tests/test_refunds.py

```diff
@@ -65,13 +65,9 @@ def test_refundable(self):
         self.assertTrue(self.enrollment.refundable())
 
     def test_refundable_expired_verification(self):
-        """ Assert that enrollment is not refundable if course mode has expired."""
+        """ Assert that enrollment is refundable if course mode has expired."""
```

Comment:
Oh, this is interesting. So I guess what we're seeing is expected behavior! Can you please let product know that on the ticket?